code/modules/reagents/newchem/chem_heater.dm

```diff
@@ -0,0 +1,118 @@
+/obj/machinery/chem_heater
```

Comment:
might as well add the board+all that jazz for this--unless that's for a future PR.